tests/10-test-nested/netconf.trace

```diff
@@ -0,0 +1,5109 @@
+**> sess:14 new session
```

Comment:
very long log file added to commit by mistake?

Reply:
Oops, sorry. Removed.